src/Query/Concerns/UsesPartitionedDml.php

```diff
@@ -18,16 +18,18 @@
 namespace Colopl\Spanner\Query\Concerns;
 
 use Colopl\Spanner\Connection;
-use Colopl\Spanner\Query\Grammar;
+use Illuminate\Database\Query\Grammars\Grammar;
 
 /**
- * @property Connection $connection
+ * @template TConnection of Connection
```

Comment:
This is not necessary, we're not passing anything other than a Spanner's connection

Reply:
Indeed. I did not think deeply when generics support in `Builder` compatible. I will put it back.